components/zcash_address/src/kind/unified/f4jumble.rs

```diff
@@ -113,15 +128,17 @@ mod tests {
 
     #[test]
     fn h_pers() {
-        assert_eq!(&H_PERS!(7), b"UA_F4Jumble_H_\x07\x00");
+        assert_eq!(&H_PERS!(7), b"UA_F4Jumble_H\x07\x00\x00");
     }
 
     #[test]
     fn g_pers() {
-        assert_eq!(&G_PERS!(7, 13), b"UA_F4Jumble_G_\x07\x0d");
+        assert_eq!(&G_PERS!(7, 13), b"UA_F4Jumble_G\x07\x0d\x00");
     }
 
```

Comment:
Relating to `f4jumble_check_vectors` below:
In https://github.com/daira/zcash-test-vectors/commit/bdf4c9ff9b4ed298bc8fa91b6e0998898d5f1ef8 I added test vectors for very large inputs, where the input is easy to generate and the output is BLAKE2b-hashed. Can you update this to check those vectors as well?

Reply:
Yup, will do, thanks for adding those.